### DISCUSSION TITLE
Only in Vehicles

### DISCUSSION THREAD

**Discussion by paulorchard** [Leawind/Third-Person#163] [Q&A] [answered]
amazing third person tool.
I'd love to be able to toggle between 1st and third with f5 still. I assume that there is a reason its.
Ideally I am working towards a game where on foot the player is in first person, but when mounted or in a vehicle they would switch to Leawind 3rd person.

**Comment by Leawind** [comment, marked as answer]
Simply use [Auto Third Person](https://modrinth.com/mod/auto-third-person)
